lib/oci/core/blockstorage_client.rb

```diff
@@ -2580,17 +2585,15 @@ def get_volume_kms_key(volume_id, opts = {})
     #   Allowed values are: TIMECREATED, DISPLAYNAME
     # @option opts [String] :sort_order The sort order to use, either ascending (`ASC`) or descending (`DESC`). The DISPLAYNAME sort order
     #   is case sensitive.
-    #
+    #    (default to ASC)
     #   Allowed values are: ASC, DESC
     # @option opts [String] :lifecycle_state A filter to only return resources that match the given lifecycle state. The state value is case-insensitive.
     #
     # @return [Response] A Response object with data of type Array<{OCI::Core::Models::BlockVolumeReplica BlockVolumeReplica}>
     # @note Click [here](https://docs.cloud.oracle.com/en-us/iaas/tools/ruby-sdk-examples/latest/core/list_block_volume_replicas.rb.html) to see an example of how to use list_block_volume_replicas API.
-    def list_block_volume_replicas(availability_domain, compartment_id, opts = {})
+    def list_block_volume_replicas(opts = {})
```

Comment:
@ZiyaoQiao was there a reason why this interface was changed?  This introduced a breaking SDK change in a minor version gem bump without appearing to have materially changed the actual API interaction.

Reply:
I don't see any mention of these breaking changes in the `CHANGELOG.md` https://github.com/oracle/oci-ruby-sdk/commit/f0198c187f0c633d86c6f3e781ee96f72c9bc1fe#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Reply:
Ruby SDK will bump the minor version if there are breaking changes. We're not releasing the Ruby SDK very often and failed to include breaking changes in the last release changelog due to the extreme long list of changes